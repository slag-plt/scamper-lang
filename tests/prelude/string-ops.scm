(make-string 5 #\a)
(make-string 0 #\c)
(string-upcase "aCcD01-E")
(string-downcase "aCcD01-E")
(string-foldcase "aCcD01-E")
(substring "hello world" 3 7)
(string->list "hello world")
(list->string (list #\h #\e #\l #\l #\o #\space #\w #\o #\r #\l #\d))