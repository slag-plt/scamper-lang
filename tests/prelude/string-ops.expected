"aaaaa"
""
"ACCD01-E"
"accd01-e"
"accd01-e"
"lo w"
(list #\h #\e #\l #\l #\o #\space #\w #\o #\r #\l #\d)
"hello world"
